src/index.ts

```diff
@@ -92,6 +92,10 @@ export class createClient {
       ...(data && body)
     })
 
+    // No JSON body if a delete response
+    if(response.status === 204)
```

Comment:
```suggestion
    if (response.status === 204)
```

Just need a space 😉 

Install Prettier to have this happen automatically on save.